Sources/PitchSpeller/Data Structures/FlowNetwork.swift

```diff
@@ -32,18 +39,78 @@ public struct FlowNetwork <Value: Hashable>: Hashable {
     /// reached zero in the flow-propagation process.
     ///
     /// - TODO: Add backflow to reversed edges.
-    public var residualNetwork: Graph<Value> {
+    public var residualNetwork: Graph<Node> {
         var residualNetwork = graph
         while let path = residualNetwork.shortestPath(from: source, to: sink) {
             residualNetwork.insertPath(path.map { $0 - maximumFlow(of: path) })
         }
         return residualNetwork
     }
+    
+    /// - Returns: (0) The maximum flow of the network and (1) the residual network produced after
+    /// pushing all possible flow from source to sink (while satisfying flow constraints) - with
+    /// saturated edges flipped and all weights removed.
+    var solvedForMaximumFlow: (Double, UnweightedGraph<Node>) {
```

Comment:
I'd label this tuple: `(flow: Double, graph: UnweightedGraph<Node>)`, or something sim., as usage later becomes cryptic otherwise.

Reply:
👍

Reply:
Nice feature, that